tests/console/python3_beautifulsoup4.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: python3-beautifulsoup tests
+# - install python-beautifulsoup package
+# - use library to parse sample htmlfile
+# - Compare the result vs expected
+#
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils 'zypper_call';
+use python_version_utils;
+use v5.20;
+use feature qw(signatures);
+no warnings qw(experimental::signatures);
+
+
+sub test_setup {
+    select_serial_terminal;
+    # Import python script and sample html file
+    assert_script_run("curl -O " . data_url("python/bs4/python3-beautifulsoup4-test.py"));
+    assert_script_run("curl -O " . data_url("python/bs4/testpage.html"));
+}
+
+sub run_test ($python_package) {
+    record_info("Testing for $python_package");
+    zypper_call("install $python_package");
+    my $python_interpreter = get_python3_binary($python_package);
+    record_info("running python version", script_output("$python_interpreter --version"));
+    assert_script_run("$python_interpreter -m venv bs4env");
+    assert_script_run("source bs4env/bin/activate");
+    assert_script_run("pip install beautifulsoup4 lxml");
```

Comment:
is this beautifulsoup and lxml downloading from pip repos? or you have already installed both packages via package manager?

otherwise the test object (packages we're shipping) won't be tested

Reply:
Good point. This is actually what [setuptools](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/python3_setuptools.pm) test is doing , but will rework to use our packages instead